security/pkg/nodeagent/caclient/providers/citadel/client.go

```diff
@@ -119,12 +121,14 @@ func (c *citadelClient) getTLSDialOption() (grpc.DialOption, error) {
 		if err != nil {
 			return nil, err
 		}
+		citadelClientLog.Warna("Citadel client using public DNS: ", c.caEndpoint)
```

Comment:
Debugf?

Reply:
I'll make it Infof - it is very hard to debug problems without this info.